### DISCUSSION TITLE
https://chat.openai.com/g/g-1QfhOlmHI

### DISCUSSION THREAD

**Discussion by giscus** [lencx/gpthub#25] [Comments]
# https://chat.openai.com/g/g-1QfhOlmHI

Discover the Best in Custom GPT at OpenAI's GPT Store – Your Adventure Begins Here!

https://gpthub.nofwl.com/

**Comment by NickL125** [comment]
最近市場如何

**Comment by justin771210** [comment]
請針對受輔導業者在模具生產製造或應用面臨問題與產業現況，
協易機械工業股份有限公司成立於1962年，主要產品有門型沖床及C型沖床兩大類，以自有品牌「SEYI」銷售客製化產品，公司為全球前五大伺服沖床製造商、全球二十大工具機設備的領先企業。公司提供從25噸到2500噸沖床的完整產品線，滿足各類型的金屬沖壓工法及不同尺寸的沖壓工件。主要客戶遍佈汽車、消費性電子、家電、建材五金、農機、航太及醫療器械的金屬零件製造產業群。2022年產品營收比重為沖床佔96%。
1.產品與技術簡介
沖床主要的用途是作為金屬材料之下料、剪斷、沖孔、折彎、抽引等工作之成型工作母機，其中以汽車業應用為大宗；少部份非金屬材料例如碳纖維、紙材或者金屬合金如鋁鎂合金，也可用沖床來加工。

2.重要原物料及相關供應商
 主要原料為機架、衝頭、平板、齒輪、曲軸、伺服馬達等，皆由各協力廠商提供或部份由國外採購。

3.主要生產據點
生產基地位於台灣桃園廠、江蘇昆山廠；2022年沖床總產能為1,483台。

(三)市場銷售與競爭
1.銷售狀況
2022年銷售地區為內銷佔8%，外銷佔92%，其中中國大陸佔44%、美洲佔28%。
 
2.國內外競爭廠商
 同業競爭部份包括國內金豐、大陸寧波米斯克及韓國最大沖床SIMPAC等製造商。

協易公司專註於伺服沖床研究與發展，伺服沖床的活用方法

伺服沖床的最大特徵，在於能夠控制衝頭的運動及速度，並配合不同的加工目的，使用最適合之沖壓曲線，達到最大的生產效率。然而，要做出最適化的沖壓曲線並非易事，我們常常碰到客戶在詢問，要如何找到這條最適當的沖壓曲線。
為找出最佳沖壓曲線，我們先將衝頭動作的目的分成兩個部分觀之。
1. 與成形加工直接相關的衝頭動作
這是模具與材料直接接觸，施加力量並進行成形的動作。這項動作的設計目的，有縮短工程、提升成形效果、提高加工精度、延長模具壽命、降低加工噪音等，目的相當多樣化。
2. 與成形加工無直接相關的衝頭動作
模具與材料接觸前或是在成形結束後，衝頭運動循環之間的接續。而衝頭動作曲線的設計目的，即在於提升產線的產能。
上述第2點的衝頭動作的最適化，因與實際的成形加工內容無直接相關，故應屬於是沖床製造商的領域。至於能夠提升多少的產能，相當大的部分要視沖床及自動化裝置的性能而定。由於各家廠商的衝頭動作曲線設計的步驟並不相同，因此洽詢自己所使用沖床的製造商尋求建議，應能獲得最佳的結果。
上述第1點的衝頭動作的最適化，則是伺服沖床最受期待的一項效果，不過要做到此點，會受到實際的製品形狀、材料性質、工程設計、設備性能等因素盤根錯節的相互影響，關於這方面系統化的研究尚未見發表。
但是依本公司與客戶的經驗交流及共同研究之下獲致了一些成果，茲將衝頭運動模式與其預期的效果整理如下，供讀者作為設計沖壓曲線時的參考。
1. 低速加工曲線
此為降低衝頭速度進行加工的曲線。這也可謂是伺服沖床一項代表性的功能。
其預期效果為：
• 降低下料加工時發生的震動及噪音
• 減少加工時的發燙、與模具耗損
• 提升成形效果
• 增加製品精度的穩定性
• 延長模具壽命
2. 持壓曲線
此為讓衝頭停止一段時間，持續進行加壓的曲線。
其預期效果為：
• 提高模具形狀的轉印性。特別適用於鋁合金、高張力鋼板等難成型素材上。
3. 多段震動曲線
此為讓衝頭快速地上下震動的同時，進行加壓加工的曲線。亦有衝頭不向上拉回的step曲線。
其預期效果為：
• 抑制抽引加工時工件產生板厚變化，並防止工件發生起皺、破裂情形。
• 使加工潤滑油的流動順暢，從而降低潤滑油的使用量。
• 減少加工時的發燙、與模具耗損
• 提高下料加工的斷面品質。
4. 壓印曲線
此為在鍛壓部分進行多次回壓的曲線。最多的案例是在下死點上使用這種曲線。
SEYI伺服沖床

為有效活用伺服沖床，在此簡單為各位介紹本公司伺服沖床的特徵。
1. 直驅式結構與專用的低轉速、高扭矩伺服馬達系統
SEYI伺服沖床的動力部構造，採用主流的直驅式結構，馬達軸直接連接驅動齒輪。且使用的沖床專用伺服馬達，是借助日本該技術領域第一人，所開發出的新型低轉速、高扭矩特性的大型伺服馬達，以及伺服控制系統，並在台灣實現了量產化。
同時兼具高出力動力源的穩定控制，與馬達動作的高速控制功能，滿足了伺服沖床所被要求的強大衝壓能力、正確且高速的衝頭動作。
2. 高剛性機架設計
隨著伺服沖床衝頭動作的進化，實現了以往難以加工的高強度材料、高精度製品的成形。SEYI伺服沖床的構造設計也因應了新世代的潮流，強化機架剛性，並採用了在複雜的衝頭動作及偏心荷重下亦能保持穩定的立體機架結構。
3. 電源平準化功能
採用符合新型伺服系統的電力組件與電容器迴路設計，將沖壓加工時與衝頭加減速食的電力負荷變化平穩化，以期降低電源容量與電費成本。

結語

本公司成立於1962年，為全球客戶提供沖床產品。自本公司於2011年發表了新世代伺服沖床SD1-200後，致力於伺服沖床的開發，提供客戶單機生產外，亦能組成連線或搭配傳送裝置，實現自動化生產線等，提供符合客戶需求的全方位解決方案。我們並提供80噸到2400噸的伺服沖床產品，支援廣泛的噸數需求。
本公司今年迎接創立56周年，未來也將持續隨著客戶的腳步，在全球推動伺服沖床產品，並以朝向更加成長為發展目標。
但協易公司於伺服沖壓模具的設計與應用了解
